### ISSUE TITLE
wrong path in source maps generated by rollup

### ISSUE THREAD

**Issue by laurmurclar** [googlemaps/react-wrapper#340]
Looks like https://github.com/googlemaps/react-wrapper/issues/336 was not fixed fully with 1.1.24. 

Below is reproducible on latest, with a fresh example app

### To reproduce
```
npx create-react-app example-app
cd example-app
npm i @googlemaps/react-wrapper
```

Change `App.js` to:
```
import React, { useEffect, useRef, ReactElement } from "react";
import ReactDOM from "react-dom";
import { Wrapper, Status } from "@googlemaps/react-wrapper";

const render = (status) => {
  if (status === Status.LOADING) return <h3>{status} ..</h3>;
  if (status === Status.FAILURE) return <h3>{status} ...</h3>;
  return null;
};

function MyMapComponent({ center, zoom }) {
  const ref = useRef();

  useEffect(() => {
    new window.google.maps.Map(ref.current, {
      center,
      zoom,
    });
  });

  return <div ref={ref} id="map" style={{ width: 800, height: 800 }} />;
}

function App() {
  const center = { lat: -34.397, lng: 150.644 };
  const zoom = 4;

  return (
    <Wrapper apiKey={process.env.REACT_APP_GOOGLE_MAPS_API_KEY} render={render}>
      <MyMapComponent center={center} zoom={zoom} />
    </Wrapper>
  );
}

export default App;
```

`npm start`
```
Compiled with warnings.

Failed to parse source map from '/Users/lauramurphy-clarkin/src/example-app/node_modules/@googlemaps/src/index.tsx' file: Error: ENOENT: no such file or directory, open '/Users/lauramurphy-clarkin/src/example-app/node_modules/@googlemaps/src/index.tsx'

src/App.js
  Line 1:36:  'ReactElement' is defined but never used  no-unused-vars
  Line 2:8:   'ReactDOM' is defined but never used      no-unused-vars

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

assets by path static/js/*.js 1.48 MiB
  asset static/js/bundle.js 1.47 MiB [emitted] (name: main) 1 related asset
  asset static/js/node_modules_web-vitals_dist_web-vitals_js.chunk.js 6.93 KiB [emitted] 1 related asset
asset index.html 1.67 KiB [emitted]
asset asset-manifest.json 458 bytes [emitted]
runtime modules 31.3 KiB 15 modules
modules by path ./node_modules/ 1.36 MiB 101 modules
modules by path ./src/ 10.1 KiB
  modules by path ./src/*.js 6.04 KiB
    ./src/index.js 1.8 KiB [built] [code generated]
    ./src/App.js 2.84 KiB [built] [code generated]
    ./src/reportWebVitals.js 1.39 KiB [built] [code generated]
  modules by path ./src/*.css 4.1 KiB
    ./src/index.css 2.72 KiB [built] [code generated]
    ./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[2]!./node_modules/source-map-loader/dist/cjs.js!./src/index.css 1.37 KiB [built] [code generated]

WARNING in ./node_modules/@googlemaps/react-wrapper/dist/index.umd.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/lauramurphy-clarkin/src/example-app/node_modules/@googlemaps/src/index.tsx' file: Error: ENOENT: no such file or directory, open '/Users/lauramurphy-clarkin/src/example-app/node_modules/@googlemaps/src/index.tsx'
 @ ./src/App.js 8:0-60 12:17-31 19:17-31 67:30-37
 @ ./src/index.js 7:0-24 11:33-36

WARNING in src/App.js
  Line 1:36:  'ReactElement' is defined but never used  no-unused-vars
  Line 2:8:   'ReactDOM' is defined but never used      no-unused-vars

1 warning has detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.69.0 compiled with 2 warnings in 6580 ms
```

Note: it looks like it's trying to find `@googlemaps/src/index.tsx` (no `react-wrapper` dir), which doesn't exist. Should it be looking for `@googlemaps/react-wrapper/src/index.tsx` instead?

**Comment by jpoehnelt** [comment]
@laurmurclar Thank you for opening this issue. 🙏
Please check out these other resources that might be applicable:

- [Support Console](https://cloud.google.com/support/) - If you have a support contract.
- [Discord](https://discord.gg/9UpV862xZ9) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag
- [Google Maps Platform issue trackers](https://developers.google.com/maps/support/#issue_tracker)


_This is an automated message, feel free to ignore._

**Comment by jpoehnelt** [comment]
looks like rollup is generating the wrong path in the source map files. not sure when I will get to debugging this.

**Comment by stevenleesf** [comment]
This problem is still there, with the latest version of react-wrapper
```
Failed to parse source map from 'C:\Users\User\Desktop\MonstaInf\paywatch\node_modules\@googlemaps\src\index.tsx' file: Error: ENOENT: no such file or directory, open 'C:\Users\User\Desktop\MonstaInf\paywatch\node_modules\@googlemaps\src\index.tsx'
```

**Comment by plgagnon2** [comment]
Is there a workaround to fix this? kinda stuck..

**Comment by jpoehnelt** [comment]
version 1.1.29 has sourcemaps disabled, please try that

**Comment by plgagnon2** [comment]
> version 1.1.29 has sourcemaps disabled, please try that

tyvm!

**Comment by shtbik** [comment]
Tried the latest CRA with [typescript](https://create-react-app.dev/docs/adding-typescript/) and still getting this issue.

```"@googlemaps/react-wrapper": "^1.1.29"```

`Failed to parse source map from '/Users/aleksandr.shtykov/Desktop/projects/hackathon/node_modules/@googlemaps/typescript-guards/src/circle.ts' file: Error: ENOENT: no such file or directory, open '/Users/aleksandr.shtykov/Desktop/projects/hackathon/node_modules/@googlemaps/typescript-guards/src/circle.ts'`

**Comment by mohamed-tayeh** [comment]
Can confirm that v.1.1.29 doesn't produce this errror

```
npm i @googlemaps/react-wrapper@1.1.29 --save
```

**Comment by shtbik** [comment]
Sorry, my mistake, issue was caused by  `"@googlemaps/typescript-guards": "^1.0.11"`, it has been fixed in [v1.0.12](https://github.com/googlemaps/js-typescript-guards/releases/tag/v1.0.12)
